analyzers/src/SonarAnalyzer.VisualBasic/SymbolicExecution/Roslyn/LocksReleasedAllPaths.cs

```diff
@@ -31,9 +31,31 @@ public class LocksReleasedAllPaths : LocksReleasedAllPathsBase
 
         protected override DiagnosticDescriptor Rule => S2222;
 
-        // ToDo: Implement early bail-out if there's no interesting descendant node in context.Node to avoid useless SE runs https://github.com/SonarSource/sonar-dotnet/issues/5375
-        public override bool ShouldExecute() =>
-            NodeContext.Node.DescendantNodes().OfType<IdentifierNameSyntax>().Any(x => ShouldExecuteFor(x.Identifier));
+        protected override ISafeSyntaxWalker GetSyntaxWalker(LockAcquireReleaseCollector collector) => new LockAcquireReleaseWalker(collector);
```

Comment:
_Nitpick:_ It's not a property and body is not trivial. Should be on the next line
```suggestion
        protected override ISafeSyntaxWalker GetSyntaxWalker(LockAcquireReleaseCollector collector) => 
            new LockAcquireReleaseWalker(collector);
```